src/core/Surface/Surface.tsx

```diff
@@ -37,33 +37,40 @@ export type SurfaceProps = {
    * Content in the surface.
    */
   children: React.ReactNode;
-} & CommonProps;
+} & Omit<CommonProps, 'title'>;
 
 /**
  * The Surface container allows content to appear elevated through the use of a drop shadow
  * @example
  * <Surface>Surface Content</Surface>
  * <Surface elevation={2}>Surface Content</Surface>
  */
-export const Surface = React.forwardRef((props: SurfaceProps) => {
-  const { elevation = 2, className, style, children, ...rest } = props;
-  useTheme();
+export const Surface = React.forwardRef<HTMLDivElement, SurfaceProps>(
+  (props: SurfaceProps, ref) => {
+    const { elevation = 2, className, style, children, ...rest } = props;
+    useTheme();
 
-  const _style = getWindow()?.CSS?.supports?.('--iui-surface-elevation: 0')
-    ? {
-        '--iui-surface-elevation': getSurfaceElevationValue(elevation),
-        ...style,
-      }
-    : {
-        'box-shadow': getSurfaceElevationValue(elevation),
-        ...style,
-      };
+    const _style = getWindow()?.CSS?.supports?.('--iui-surface-elevation: 0')
```

Comment:
Ideally, would need to check with `getSurfaceElevationValue`, not 0, but I guess minor. Can replace with `none`, for consistency.

Reply:
Changed to include `getSurfaceElevationValue`

`--iui-surface-elevation: ${getSurfaceElevationValue(elevation)},`